examples/react-client/prettier-chat/vite.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react-swc";
+import path from "path";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
```

Comment:
suggestion: I would set the default port to, for example, 3010 to make a clear distinction from the old app

Reply:
d